app/components/developers/card_component.html.erb

```diff
@@ -15,11 +15,14 @@
         </div>
       </div>
 
-      <% if featured? %>
-        <div class="mt-1 pb-1">
-          <span class="inline-flex items-center px-2.5 py-0.5 rounded-md text-sm font-medium bg-blue-100 text-blue-800"> <%= t(".featured") %> </span>
-        </div>
-      <% end %>
+     <div class="mt-1 pb-1">
+        <% if featured? %>
+          <%= render Developers::BadgeComponent.new(t(".featured"), classes: "inline-flex items-center rounded-md px-2.5 py-0.5 text-sm font-medium bg-blue-100 text-blue-800") %>
```

Comment:
Then, down here we could do:

```ruby
<%= render BadgeComponent.new(t(".featured"), color: :blue) %>
```

And on line 23:

```ruby
<%= render BadgeComponent.new(t(".featured"), color: :green) %>
```

Reply:
Thanks. Will make required changes in the code.